libs/components/http/src/lib/modules/auth-http-client/auth-token-provider.spec.ts

```diff
@@ -60,49 +60,107 @@ describe('Auth token provider', () => {
   });
 
   describe('getContextToken() method', () => {
-    it("should call BBAuth.getToken() with the SPA's context parameters", () => {
-      const getTokenSpy = spyOn(BBAuthClientFactory.BBAuth, 'getToken');
+    describe("should call BBAuth.getToken() with the SPA's context parameters", () => {
```

Comment:
The way the expectations are written in the describe/it calls reads awkwardly. You should be able to concatenate all the nested describe/it expectations and have one coherent sentence, since this is what the Karma test reporter does when it reports a failure. Right now it would read "getContextToken() method should call BBAuth.getToken() with the SPA's context parameters include environment id". 

The `describe` expectation should be a thing or a condition while `it` should start with `should` followed by the expected action. Something like this would work better (which would result in something like "getContextToken() method when calling BBAuth.getToken() with the SPA's context parameters should include environment ID"):

```
describe('getContextToken() method', () =>  {
  describe('when calling BBAuth.getToken() with the SPA's context parameters', () => {
    it('should include environment ID', () => {
    });

    it('should include legal entity ID', () => {
    });
  });
});
}
```

Reply:
Updated 👍